### ISSUE TITLE
Crash with a textbox 'hidden' in a form

### ISSUE THREAD

**Issue by fraisse** [gcanti/tcomb-form#284]
I think you have introduced a regression in your last release removing `uvdom`.
Rendering a hidden textbox returns an object with a tag which has to be compiled with `uvdom`:
```javascript
return {
      tag: 'input',
      attrs: {
        type: 'hidden',
        value: locals.value,
        name: locals.name
      }
    }
```
But it is not compiled anymore.
Then react complains: `Uncaught Invariant Violation: Textbox.render(): A valid ReactComponent must be returned. You may have returned undefined, an array or some other invalid object.`

**Comment by gcanti** [comment]
Thanks @fraisse, working on this...

**Comment by gcanti** [comment]
Just released a fix in https://github.com/gcanti/tcomb-form-templates-bootstrap/releases/tag/v0.1.1
Reinstalling the package `tcomb-form-templates-bootstrap` should fix the problem

**Comment by fraisse** [comment]
Thanks a lot for your reactivity. It works like a charm with this new tcomb-form-templates-bootstrap release ;-)

**Comment by gcanti** [comment]
Great, thanks for the feedback ;)

**Comment by fraisse** [comment]
thanks for your work ;-)

**Comment by fraisse** [comment]
Hi Giulio,

I know this bug is fixed but it is fixed in the new version of the `tcomb-form-templates-bootstrap` library you forget to update in your present dependencies. Have you planned to update your dependencies?

**Comment by gcanti** [comment]
Hi @fraisse!

> you forget to update in your present dependencies

Not sure I'm follow. In `package.json` there is `"tcomb-form-templates-bootstrap": "^0.1.0"`. So when you manage a fresh install of `tcomb-form` npm should dowload the fixed version (`0.1.1`). Am I missing something?

**Comment by fraisse** [comment]
For a fresh install you right but it is not true on an existing install with a version `0.1.0` of `tcomb-form-templates-bootstrap` which is compatible with `^0.1.0`.
